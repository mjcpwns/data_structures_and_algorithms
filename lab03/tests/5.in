10 5 14 30 29 32 24 7 9 3 26