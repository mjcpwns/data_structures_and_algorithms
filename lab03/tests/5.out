Rotating left
Rotating left
Rotating left
Rotating left
Rotating right
Rotating left
Tree:
       10
       / \
      /   \
     /     \
    7      29
   / \     / \
  5   9   /   \
 /       24   30
3       / \     \
       /   \    32
      14   26

#nodes = 11,  #leaves = 5
Infix  : 3 5 7 9 10 14 24 26 29 30 32 
Prefix : 10 7 5 3 9 29 24 14 26 30 32 
Postfix: 3 5 9 7 14 26 24 32 30 29 10 
ByLevel: 10 7 29 5 9 24 30 3 14 26 32 
